api.planx.uk/send/exportZip.ts

```diff
@@ -109,17 +109,22 @@ export async function buildSubmissionExportZip({
     }
   }
 
+  const boundingBox = passport.data["property.boundary.site.buffered"];
```

Comment:
This is hardcoded on a user-based variable - which I think is realistically fine? 

Alternatively we could hardcode to it a dev-based variable (e.g. `passport.data._somethingWeChose`)

Reply:
Agree this is realistically fine! Can become a variable we validate existence of on publish in relation to future integrations/schema work :slightly_smiling_face: